tests/bluetooth/audio/cap_commander/src/test_distribute_broadcast_code.c

```diff
@@ -240,6 +240,7 @@ ZTEST_F(cap_commander_test_distribute_broadcast_code,
 			   mock_cap_commander_distribute_broadcast_code_cb_fake.call_count);
 }
 
+#if CONFIG_BT_MAX_CONN >= 2
```

Comment:
Suggest to use same approach as you did here https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bluetooth/audio/cap_commander/src/test_broadcast_reception.c#L424-L426

Reply:
Oops, forget about that way of doing it.